cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -488,9 +488,8 @@ def create(self, catalog_item_class, *args, **kwargs):
         view.fill(cat_item.fill_dict)
         view.add.click()
         view = self.create_view(AllCatalogItemView)
-        # TODO move this assertion to tests
-        # view.flash.assert_success_message('Catalog Item "{}" was added'.format(
-        #     cat_item.name), partial=True)
+        view.flash.assert_success_message('Catalog Item "{}" was added'.format(
```

Comment:
Required:
`L494` checks `no_error` so as per our practice. we should move this `flash` msg assertion to the respective test. what do you think?